dashboard/test/models/pd/workshop_test.rb

```diff
@@ -63,6 +63,12 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
     assert_equal workshops.first, @workshop
   end
 
+  test 'exclude_summer scope' do
+    summer_workshop = create :pd_workshop, :local_summer_workshop
+
+    assert Pd::Workshop.exclude_summer.exclude? summer_workshop
```

Comment:
ideally, create 3 workshops: a local summer, teachercon, and something else that should be included. Verify the 2 summers are excluded and the other is included